google/auth/identity_pool.py

```diff
@@ -117,28 +120,65 @@ def __init__(
                     )
             else:
                 self._credential_source_field_name = None
-        else:
-            self._credential_source_file = None
-        if not self._credential_source_file:
-            raise ValueError("Missing credential_source file")
+
```

Comment:
Could you also provide an example `credentials_source` in the function docstring so it's easy for folks to figure out the expected format?

Reply:
Done.  I added a couple of different examples